### ISSUE TITLE
the cursor jumps backwards when browsing certain websites in Firefox, Chrome and Edge

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#7956]
### Steps to reproduce:
1. Open this website in Edge or Firefox
https://groups.google.com/forum/#!topic/braillists/VharTVIER98
2. For a comparable example in google Chrome see issue #7955 or directly this website:
https://www.zhihu.com/question/58178068
3. Navigate in browse mode by using down and up arrow
4. Navigate in focus mode using tab and shift+tab

### Expected behavior:
NVDA should allow a stable navigation without changing cursor position.

### Actual behavior:
The position is changing very often and the cursor moves backwards automatically.

### System configuration:
NVDA version:
2017.4

NVDA Installed or portable:
Both

Other information:

Windows version:
Windows 10.16299.64

Name and version of other software in use when reproducing the issue:
Firefox 58, MS Edge, Google Chrome 64

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
no.

**Comment by PratikP1** [comment]
Can you provide the exact text you hear and you expect to hear when tabbing? As far as I can tell from my testing in Firefox and Edge, this page seems to follow the layout of the page as set by the developers. What exactly do you wish NVDA did differently in this case or cases like these?

**Comment by Adriani90** [comment]
Hello, let's take the google groups page in Firefox. When you go to the message region and reach the first profile, NVDA say "Lora's profile foto". But when you press down arrow in browse mode to continue reading, NVDA jumps back to "Lora's profile foto". You have to press multiple times down arrow to get out of that region and begin reading the message. Sometimes NVDA jumps backwards three of four times until it gets the right position. Especially when you navigate slowly. If you open that page in google chrome or Internet Explorer, you will see the difference.

The same happens when you open the second link in google chrome, but that pchinese page works well in Firefox and IE or Edge.


Now when you activate focus mode in Firefox to navigate the google groups page, following happens:
* Go to Lora's profile foto and press NVDA+space bar to activate focus mode
* Press Shift+ tab and then tab again.

You will notice that NVDA gets stacked on the element g+. Sometimes in focus mode NVDA jumps backwards.


To summarize, navigating some websites with NVDA is not very stable. Take this page as a comparison. This website works very stable in every browser:
https://www.wired.com/

**Comment by Brian1Gaff** [comment]
I've been having issues with all versions of firefox on the filehippo update 
check page, ie what loads when you run their program. it runs firefox and 
then displays a page with some clickable options and the list of software it 
thinks are out of date.
 Sadly this works only about 1 time in 10 recently even with the esr version 
of Firefox. Either its there and you can access the clickable and the items 
in the list or next time its vanished and the clickable look like just text 
and do not click. No amount of refreshing makes tit work but it always works 
in IE11.
 Brian

**Comment by Adriani90** [comment]
#8013

**Comment by Adriani90** [comment]
This is related to 
#2039

**Comment by Adriani90** [comment]
This issue exists only in MS Edge. It is no longer occuring in Firefox and Google Crome, neither in IE. Tested with NVDA 2018.4 Beta 4 and Firefox 63.0.3, IE11 and Google Crome 71.0. Therefore I am going to close it and reopen it if I find an other example to reproduce this issue.
